signal-hook-registry/src/lib.rs

```diff
@@ -247,7 +334,12 @@ fn without_signal<F: FnOnce() -> Result<(), Error>>(signal: c_int, f: F) -> Resu
 /// these signals is attempted.
 ///
 /// See [`register`](fn.register.html).
-pub const FORBIDDEN: &[c_int] = &[SIGKILL, SIGSTOP, SIGILL, SIGFPE, SIGSEGV];
+pub const FORBIDDEN: &[c_int] = _FORBIDDEN;
```

Comment:
Why is there the indirection? To reuse the doc string?

I'd probably prefer to use something like `FORBIDDEN_IMPL` or `FORBIDDEN_INNER` instead of surrounding names with underscores.